backend/hitas/calculations/max_price.py

```diff
@@ -131,20 +135,35 @@ def calculate_max_price(
                 "city": apartment.postal_code.city,
             },
             "ownership": [
```

Comment:
I think this should be `ownerships` (instead of `ownership`), as it may contain multiple values and to be consistent with other code

Reply:
fixed